client/my-sites/checklist/wpcom-checklist/checklist-navigation/index.jsx

```diff
@@ -25,64 +24,24 @@ export class ChecklistNavigation extends Component {
 	static propTypes = {
 		siteSlug: PropTypes.string,
 		translate: PropTypes.func.isRequired,
-		setStoredTask: PropTypes.func.isRequired,
-		setNotification: PropTypes.func.isRequired,
-		showNotification: PropTypes.bool,
-		closePopover: PropTypes.func.isRequired,
 		recordTracksEvent: PropTypes.func.isRequired,
 	};
 
-	handleClick = () => {
-		const task = this.props.taskList.getFirstIncompleteTask();
-
-		if ( task ) {
-			this.props.setStoredTask( task.id );
-		}
-		this.props.setNotification( false );
-		this.props.closePopover();
-		this.props.recordTracksEvent( 'calypso_inlinehelp_checklist_click' );
-	};
-
 	render() {
-		const { isLoading, siteSlug, translate, showNotification, taskList } = this.props;
+		const { isLoading, translate, taskList } = this.props;
 		const { total, completed } = taskList.getCompletionStatus();
 
 		if ( total === completed || isLoading ) {
 			return null;
 		}
 
-		const buttonClasses = classNames( 'checklist-navigation__count', {
-			'has-notification': showNotification,
-		} );
-
-		const checklistLink = '/checklist/' + siteSlug;
-
 		return (
 			<div className="checklist-navigation">
-				<Button
-					onClick={ this.handleClick }
-					href={ checklistLink }
-					className="checklist-navigation__button"
-					borderless
-				>
-					<span className="checklist-navigation__label">
-						{ translate( 'Continue Site Setup' ) }
-					</span>
-
-					<span className={ buttonClasses }>
-						{ translate( '%(complete)d/%(total)d', {
-							comment: 'Numerical progress indicator, like 5/9',
-							args: {
-								complete: completed,
-								total: total,
-							},
-						} ) }
-					</span>
+				<span className="checklist-navigation__label">{ translate( 'Finish setup' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Finish Setup' )` *ES Score: 12*